client-app/src/desktop/common/Wrapper.js

```diff
@@ -13,12 +13,10 @@ import './Wrapper.scss';
 export const [Wrapper, wrapper] = hoistCmp.withFactory({
     displayName: 'Wrapper',
     className: 'tbox-wrapper xh-tiled-bg',
-    model: false,
-
-    render({className, description, links, children, ...rest}) {
-
-        const localModel = useLocalModel(() => new Model(links));
+    render({className, description, children, ...props}) {
+        const impl = useLocalModel(Model);
```

Comment:
Should this be "WrapperModel". Also why does this one need to use "useLocalModel"

Reply:
yes, and yes.  good suggestions